tools.js

```diff
@@ -311,6 +311,7 @@ module.exports = (function () {
 				else move.gen = 0;
 			}
 			if (!move.priority) move.priority = 0;
+			if (!move.ignoreImmunity) move.ignoreImmunity = (move.category === 'Status' ? true : {});
```

Comment:
It messes up Scrappy, I guess? But I guess Scrappy can initialize the object itself instead (this will means I can probably take it out of the `Tools#getMove`)